ui/webui/src/components/app.jsx

```diff
@@ -104,6 +108,12 @@ export const Application = () => {
         if (prevStep.prevId === "installation-destination") {
             applyDefaultStorage({ address, onAddErrorNotification, onSuccess: () => setIsStorageReady(true) });
         }
+        const stepIdx = steps.findIndex(s => s.id === stepIdReached.id);
+        const newStepIdx = steps.findIndex(s => s.id === newStep.id);
```

Comment:
`stepIdReached` should not use `.id` here, I think? Below you do `setStepIdReached(newStep.id);` which already does that part.

Reply:
Oh great catch!!

Reply:
Fixed! thanks for noticing!